app.py

```diff
@@ -60,6 +61,9 @@ def sync_team(client=None, owner=None, team_id=None, slug=None):
     custom_map = load_custom_map()
     try:
         directory_group = custom_map[slug] if slug in custom_map else slug
+        if SYNCMAP_ONLY and slug not in custom_map:
+            print("skipping team {0} - not in sync map".format(slug))
+            return
```

Comment:
@tombokombo while this would certainly work this way, larger directories will needlessly traverse the user directory. It would be better to put this logic in the main `sync_all_teams` method to keep API calls to a minimum

https://github.com/github/github-team-sync/blob/main/app.py#L279-L286

Reply:
sure, already moved,  check now please